github/processors/bodyMarkdowmRenderer.ts

```diff
@@ -0,0 +1,112 @@
+import {
+    BlockBuilder,
+    TextObjectType,
+} from "@rocket.chat/apps-engine/definition/uikit";
+
+
+function cleanHeadingSyntax(text: string) : string{
+    try {
+        text = text.replace(/(#{3}\s)(.*)/g, (val) => `*${val.substring(3, val.length).trim()}*`);
+        text = text.replace(/(#{2}\s)(.*)/g, (val) => `*${val.substring(3, val.length)}*`);
+        text = text.replace(/(#{1}\s)(.*)/g, (val) => `*${val.substring(2, val.length)}*`);
+        text = text.replace(/\[ ] (?!\~|\[\^\d+])/g, (val) => `⭕ ${val.substring(3, val.length)}*`);
+        text = text.replace(/\[X] (?!\~|\[\^\d+])/g, (val) => `✅ ${val.substring(3, val.length)}*`);
+    }
+    catch(e){
+        console.log(e);
+    }
+
+    return text;
+}
+
+
+export async function BodyMarkdownRenderer({
```

Comment:
@henit-chobisa  No change, just wanted to thank you for working on this !! Really appreciate your work with the markdown rendering :clap: :clap: :smile:

Reply:
Thank you so much, Samad,
I love the concept of the "GitHub App" actually and I think if we work harder on this, we can make it stand in those amazing features of RC, that make customers helpless to switch from their existing platforms to RC. 😋